testo.d/lowranceusr.test

```diff
@@ -1,36 +1,150 @@
-
 #
-# Lowrance USR.  Binary, and also slightly lossy because of the math to
-# convert lat/long.  It also doesn't support description, which makes it
-# awkward  to test.
+# Lowrance USR
+# ============
+# 
+# Binary data format, and slightly lossy because of the math to
+# convert lat/long.
+#
+# File Name		  	File Contents
+# ===============		========================
+# lowrance.usr			USR version 2 format file.  Generated by GPSBabel from geocaching.loc input..
```

Comment:
Its very nice to document where reference files came from, and which are golden, i.e. directly from the device.  They way we store reference files and translations of reference files in the same directory is confusing at best.